Jenkinsfile

```diff
@@ -1,29 +1,40 @@
 pipeline {
     agent any
-    
+
     stages {
+        stage('Install Deps') {
+            steps {
+                bat 'npm install'
+            }
+        }
+        stage('Lint') {
+            steps {
+                bat 'npm run lint-fix'
+            }
+        }
         stage('Build') {
             steps {
-                bat 'echo Building...'
+                bat 'git reset --hard'    /* do this to avoid unstaged changes error */
+                bat 'git rebase origin/%CHANGE_TARGET%'
+                bat 'npm run build'
             }
         }
         stage('Test') {
             steps {
-                bat 'echo Testing...'
-                bat 'echo Multiline shell steps work too'
-                bat 'echo This is another line'
+                bat 'npm run unit'
+                // bat 'npm run e2e'
```

Comment:
This is commented because it is currently broken but there is a [PR](https://github.com/371team4/371Team4/pull/19) for the fix